packages/govuk-frontend/src/govuk-prototype-kit/init.scss

```diff
@@ -2,7 +2,7 @@ $govuk-prototype-kit-major-version: 0 !default;
 
 $govuk-assets-path: if(
   $govuk-prototype-kit-major-version > 12,
-  $govuk-extensions-url-context + "/govuk-frontend/govuk/assets/",
+  $govuk-extensions-url-context + "/govuk-frontend/dist/govuk/assets/",
   "/govuk/assets/"
```

Comment:
No change to make for earlier versions of the Prototype Kit, as [the kit seems to mount the GOV.UK Frontend assets under `/govuk/assets`](https://github.com/alphagov/govuk-prototype-kit/blob/4770bcfcb069368809ef77c8f3e189e51c7e657a/app/assets/sass/application.scss#L6).